server/services/temporaryAbsencesService.ts

```diff
@@ -0,0 +1,20 @@
+import type { TemporaryAbsence } from 'welcome'
+import type { RestClientBuilder, WelcomeClient, HmppsAuthClient } from '../data'
+
+export default class TemporaryAbsencesService {
+  constructor(
+    private readonly hmppsAuthClient: HmppsAuthClient,
+    private readonly welcomeClientFactory: RestClientBuilder<WelcomeClient>
+  ) {}
+
+  private compareByName(a: TemporaryAbsence, b: TemporaryAbsence): number {
+    const result = a.lastName.localeCompare(b.lastName)
+    return result !== 0 ? result : a.firstName.localeCompare(b.firstName)
```

Comment:
cool, next step is to create a fancy generic method version. Can talk about it tomorrow